deployment/terraform/engine.go

```diff
@@ -85,3 +86,16 @@ func (t *Terraform) runCommand(dst io.Writer, args ...string) error {
 	}
 	return nil
 }
+
+func checkTerraformVersion() error {
+	out, err := exec.Command("terraform", "version").Output()
+	if err != nil {
+		return fmt.Errorf("could not run terraform version command: %w", err)
+	}
+	if re := regexp.MustCompile(`v\d+.?\d+`); !re.Match(out) {
+		return fmt.Errorf("could not parse terraform command output: %s", out)
+	} else if version := string(re.FindAll(out, 1)[0]); version != "v0.12" {
```

Comment:
Hmm .. this will break when 0.13 is released. Are we sure we want to pin down to a version ?

Also, we can use FindAll itself to confirm whether it matched or not. No need to make 2 calls.

Reply:
I suggested this cause in the docs we explicitly say we support v0.12 and we have no guarantees that when v0.13 comes out everything will work out of the box. So it was about being consistent with documentation and avoid more confusion for the users.